intranet/static/js/eighth/signup.search.js

```diff
@@ -195,15 +190,16 @@ $(document).ready(function() {
                     if (queries[i] === "or") {
                         //console.debug("OR:", i);
                         i = parseInt(i);
-                        if (i-1 >= 0 && i+1 < queries.length) {
+                        if (i - 1 >= 0 && i + 1 < queries.length) {
                             //console.debug("orqs:", queries[i-1], queries[i+1]);
                             //console.debug("orqs:", shows[i-1], shows[i+1]);
-                            nshows.push(shows[i-1] || shows[i+1]);
+                            nshows.push(shows[i - 1] || shows[i + 1]);
                         }
                     }
                 }
+
                 for (i in nshows) {
-                    if (nshows[i]) {show = true;}
+                    if (nshows[i]) show = true;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["i" used outside of binding context.](https://www.codacy.com/app/tjcsl/ion/file/2755451097/issues/source?bid=3176160&fileBranchId=3355598#l202)